{{cookiecutter.project_name}}/notebooks/ModelValidation.py

```diff
@@ -0,0 +1,224 @@
+evaluators = "default"
+assert env != "None", "env notebook parameter must be specified"
+assert model_uri != "", "model_uri notebook parameter must be specified"
+assert model_name != "", "model_name notebook parameter must be specified"
+assert model_version != "", "model_version notebook parameter must be specified"
+
+# set experiment
+with open("../databricks-config/output/prod.json", "r") as f:
+    experiment_name = json.load(f)["{{cookiecutter.project_name}}_experiment_name"]["value"]
+mlflow.set_experiment(experiment_name)
+
+# COMMAND ----------
+
+##################################################################################
+# TODO : Please fill in this cell with proper values for enable_baseline_comparison,
+#        data, targets, model_type, custom_metrics, validation_thresholds
+#        evaluator_config
+##################################################################################
+
+# Whether to load the current registered "Production" stage model as baseline. A version with "Production" stage must 
+# exist for the model.
+# Baseline model is a requirement for relative change and absolute change validation rules.
+# TODO(required) : enable_baseline_comparison
+enable_baseline_comparison = False
+
+# model validation data input for staging or prod workspace. A Pandas DataFrame or Spark DataFrame, containing evaluation features and labels.
+# Please refer to data parameter in mlflow.evaluate documentation https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.evaluate
+if env == "prod":
+    # TODO(required) : set data for prod workspace
+    data = None
+elif env == "staging":
+    # TODO(required) : set data for staging workspace
+    data = None
+else:
+    raise Exception("Unknown environment. Please select 'prod' or 'staging' as environment name")
+
+# The string name of a column from data that contains evaluation labels.
+# Call get_targets_from_recipe() to get targets from recipe configs if mlflow recipe is used. 
+# Please refer to targets parameter in mlflow.evaluate documentation https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.evaluate
+# TODO(required) : targets
+targets = get_targets_from_recipe()
+
+
+# A string describing the model type. The model type can be either "regressor" and "classifier".
+# Call get_model_type_from_recipe() to get model type from recipe configs if mlflow recipe is used. 
+# Please refer to model_type parameter in mlflow.evaluate documentation https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.evaluate
+# TODO(required) : model_type
+model_type = get_model_type_from_recipe()
+
+# Custom metrics to be included. Set it to None if custom metrics are not needed.
+# Please refer to custom_metrics parameter in mlflow.evaluate documentation https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.evaluate
+# TODO(optional) : define custom metric function as necessary
+# TODO(optional) : custom_metrics
+custom_metrics = []
+
```

Comment:
I think you should try to read from recipes for custom metrics, if possible.

Reply:
For custom_metrics we should read from recipe if possible. For thresholds there's problem of whether we want to just import the conditions or allow overwrite. Also recipe thresholds doesn't support baseline comparison. 
How about let's leave this part out of the PR for now and see what users prefer later.